flocker_bb/builders/flocker_vagrant.py

```diff
@@ -218,6 +218,40 @@ def buildTutorialBox():
     return factory
 
 
+def run_client_installation_tests(configuration):
+    """Run the installation tests for clients."""
+    factory = getFlockerFactory()
+
+    if configuration.provider == 'vagrant':
```

Comment:
I'm confused by this - it reads that a step is added to destroy a box. Expand the comment to say what this does and not just why it is done.